src/main/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysis.java

```diff
@@ -225,10 +252,18 @@ private SecurityAnalysisResult runSimulations(LfNetwork network, List<Propagated
                                                   LoadFlowParameters loadFlowParameters, OpenLoadFlowParameters openLoadFlowParameters) {
         // create a contingency list that impact the network
         List<LfContingency> contingencies = createContingencies(propagatedContingencies, network);
+        List<BranchResult> preContingencyBranchResults = new ArrayList<>();
+        List<BusResults> preContingencyBusResults = new ArrayList<>();
 
         // run pre-contingency simulation
         try (AcloadFlowEngine engine = new AcloadFlowEngine(network, acParameters)) {
             AcLoadFlowResult preContingencyLoadFlowResult = engine.run(Reporter.NO_OP);
+            monitors.values().forEach(stateMonitor -> {
+                if (stateMonitor.getContingencyContext().getContextType() == ContingencyContextType.ALL ||
```

Comment:
@sylvlecl Reading again this, I am not sure that our choice on `powsybl-core` side is the good one...

Reply:
You mean to include the "ALL" into the N situation ?